frontend/src/hooks/useApi.ts

```diff
@@ -1,4 +1,4 @@
-import CustomAxios from 'customAxios';
+import CustomAxios, { LifecycleToasts } from 'customAxios';
```

Comment:
Can you extract all your changes to a separate file called `useApiAdminAreas.ts`?  I'm in the middle of a massive refactor to support updates to projects.  This file is going to disappear sooner than later.

Reply:
Yup will do that!